src/overlay/StellarXDR.h

```diff
@@ -9,4 +9,5 @@ namespace stellar
 {
 
 std::string xdr_printer(const PublicKey& pk);
+std::string xdr_printer(const MuxedAccount& ma);
```

Comment:
Nitpick: we use the consistent-const convention (`const` is on the right always). See https://github.com/stellar/stellar-core/blob/master/CONTRIBUTING.md#code-style. In this case, the existing code already wasn't using our convention so if you could update both overloads of `xdr_printer` that would be great.